From 31c4f557eebd54630d276d4337edaed60b18db76 Mon Sep 17 00:00:00 2001
From: Antonio Borneo <antonio.borneo@foss.st.com>
Date: Wed, 21 Jun 2023 10:45:04 +0200
Subject: [PATCH 310/841] feat(stm32mp2): update compatible string for watchdog

The compatible string for STM32MP25x watchdog is the same as for
the STM32MP1xx devices.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/arm-trusted-firmware.git 71a4c69b85102bd6cf91c5ca6ddaae3223d0cd9c feat(stm32mp2): update compatible string for watchdog]
Signed-off-by: Antonio Borneo <antonio.borneo@foss.st.com>
Change-Id: Ib55a95e4e05e427f9216558591bd877b839b6abc
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/tf-a/+/314039
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
Reviewed-by: Patrick DELAUNAY <patrick.delaunay@foss.st.com>
Reviewed-by: Yann GAUTIER <yann.gautier@foss.st.com>
Domain-Review: Yann GAUTIER <yann.gautier@foss.st.com>
---
 plat/st/stm32mp2/stm32mp2_def.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/plat/st/stm32mp2/stm32mp2_def.h b/plat/st/stm32mp2/stm32mp2_def.h
index a8a1cd43e..f98c79725 100644
--- a/plat/st/stm32mp2/stm32mp2_def.h
+++ b/plat/st/stm32mp2/stm32mp2_def.h
@@ -529,7 +529,7 @@ static inline uintptr_t tamp_bkpr(uint32_t idx)
  ******************************************************************************/
 #define DT_BSEC_COMPAT				"st,stm32mp25-bsec"
 #define DT_DDR_COMPAT				"st,stm32mp2-ddr"
-#define DT_IWDG_COMPAT				"st,stm32mp2-iwdg"
+#define DT_IWDG_COMPAT				"st,stm32mp1-iwdg"
 #define DT_MMIO_SRAM				"mmio-sram"
 #define DT_PWR_COMPAT				"st,stm32mp25-pwr"
 #define DT_RCC_CLK_COMPAT			"st,stm32mp25-rcc"
-- 
2.47.2

